fasthtml/core.py

```diff
@@ -694,7 +695,9 @@ def __getattr__(self, name):
 
     def to_app(self, app):
         "Add routes to `app`"
-        for args in self.routes: app._add_route(*args)
+        for args in self.routes: 
+            print(args)
```

Comment:
```suggestion

```

Reply:
yah was hoping to get that fixed before you reviewed :). It should be good now.